apps/blockchain/test/blockchain_test.exs

```diff
@@ -21,6 +21,9 @@ defmodule BlockchainTest do
     ],
     "Byzantium" => String.split(@failing_byzantium_tests, "\n"),
     "Constantinople" => [
+      "GeneralStateTests/stCreate2/create2collisionStorage_d0g0v0.json",
```

Comment:
I don't think so because it's not included in the [list of included EIPs in Constantinople
](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1013.md#specification).

I don't think there are tests for this case before the `Constantinople` fork.

It is odd this test was passing before though. I made [a separate issue](https://github.com/poanetwork/mana/issues/442) to look into what's going on here.